app/config/AppConfig.scala

```diff
@@ -58,10 +58,11 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig) {
   lazy val signInUrl: String = s"$signInBaseUrl?continue=$signInContinueUrlRedirect&origin=$signInOrigin"
   lazy val additionalInformationServiceBaseUrl: String = s"${servicesConfig.getString(additionalInformationUrlKey)}/income-tax-additional-information"
 
-  lazy val useEncryption: Boolean = servicesConfig.getBoolean("useEncryption")
+  lazy val newGainsServiceEnabled: Boolean = servicesConfig.getBoolean(key = "feature-switch.newGainsServiceEnabled")
```

Comment:
Could this have a better name?  Along the lines of 'backendSessionEnabled' or something that describes the purpose of the feature switch more clearly.